### PR TITLE
Variance alias support

### PR DESCRIPTION
**Pull request by gmathieu**
### Proposed changes

Add Alias method docs now that Variance supports it.

### Merge timing

ASAP 🙏

### CODE REVIEW

**Comment by gmathieu** [comment]
Hi @stayseesong,

I'm waiting on Segment's engineering team to update our integration. Once updated, I'll submit this PR for review. Here's the [ticket](https://segment.zendesk.com/hc/requests/473200) if you have access.

Thanks for following up

File: src/connections/destinations/catalog/variance/index.md [hunk 1]
```diff
@@ -85,6 +85,16 @@ analytics.group("groupId123", {
 
 Segment sends Group calls to Variance as an `Account` if you've chosen the "Group" Account Mapping method during setup.
 
+### Alias
+
+If you aren’t familiar with the Segment Spec, take a look at the [Alias method documentation](/docs/connections/spec/alias/) to learn about what it does. An example call would look like this:
```
Comment (stayseesong) [inline src/connections/destinations/catalog/variance/index.md:90]:
```suggestion
If you're not familiar with the Segment Spec, take a look at the [Alias method documentation](/docs/connections/spec/alias/) to learn about what it does. An example call would look like this:
```

File: src/connections/destinations/catalog/variance/index.md [hunk 2]
```diff
@@ -85,6 +85,16 @@ analytics.group("groupId123", {
...
+
+```js
+analytics.alias('123456')
+```
+
+If there is an existing user with that `userId` in Variance that matches the `previousId` passed with the alias that user will be merged into the "123456" user identified in the alias. If there is only one user with the `previousId` and no user with the `userId` that user will simply have their Variance `externalId` updated to match the current `userId` passed in the alias.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/variance/index.md:96]:
```suggestion
If there is an existing user with that `userId` in Variance that matches the `previousId` passed with the alias, that user will be merged into the "123456" user identified in the alias. If there is only one user with the `previousId` and no user with the `userId`, that user will have their Variance `externalId` updated to match the current `userId` passed in the alias.
```

**Review by stayseesong — DISMISSED** [review]

**Comment by gmathieu** [comment]
@stayseesong I addressed your feedback. Thanks!

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**